### ISSUE TITLE
No lastvalue for manually added device

### ISSUE THREAD

**Issue by Jean-PhilippeD** [node-enocean/node-enocean#38]
Hello,

I'm trying to use this node module for my enocean devices but it's quite hard for me to understand how to work with.

I'm using a web app (Gladys project) and trying to integrate node-enocean as a plugin.

First of all, I see that I need to save/dump device on a json file, that's what I successfully did. By the way once device are learned, I'll need to save it in mysql db.

As I move step by step, I see that when I've learned device by hand, I see the device are "known" but the last values are not returned correctly : 

```
last value: { timestamp: 1509808560835,
  rawByte: '55000707017af6000190503b2004ffffffff5600fc',
  packetType: 1,
  senderId: '0190503b',
  choice: 'f6',
  subTelNum: 4,
  destinationId: 'ffffffff',
  rssi: 86,
  securityLevel: 0,
  raw: '00',
  packetTypeString: 'RPS',
  sensor: 
   { id: '0190503b',
     eep: 'd2-01-0a',
     desc: 'actuator',
     manufacturer: 'Nodon' },
  values: [ { type: 'unknown', unit: 'unknown', value: 'unknown' } ] }

```

What am I doing wrong ?

**Comment by Holger-Will** [comment]
yes, this is a known issue. the nodon modules advertise themselfs as sending d2-01-0x but also are sending RPS (f6-02-01) telegrams via the same address. 

Usually if a device is sending multible different telegram types it should do so using multible different addresses... which nodon does not do... i still don't know how to handle that case properly. 

I'm open for any ideas.

**Comment by Holger-Will** [comment]
#32

**Comment by Jean-PhilippeD** [comment]
Ok I understand, thanks for the quick reply.

But I might do something wrong because for a "contact device", I have the same issue:
```

last value: { timestamp: 1509811924272,
  rawByte: '55000707017ad509018029c10006ffffffff56006b',
  packetType: 1,
  senderId: '018029c1',
  choice: 'd5',
  subTelNum: 6,
  destinationId: 'ffffffff',
  rssi: 86,
  securityLevel: 0,
  raw: '09',
  learnBit: 1,
  packetTypeString: '1BS',
  sensor: 
   { id: '018029c1',
     eep: 'D5-00-01',
     desc: 'Door contact',
     manufacturer: 'Nodon' },
  values: [ { type: 'unknown', unit: 'unknown', value: 'unknown' } ] }
```

**Comment by Jean-PhilippeD** [comment]
hum just added a temperature sensor and I get correct value !

`[ { type: 'Temperature', unit: '°C', value: 21.019607843137255 } ]
`

So what's happening with my contact sensor

**Comment by Holger-Will** [comment]
right, the problem is realy just the nodon module... for the multible telegram problem, i might just check the telegram types and save every different type to the history store. I'll see3 what i can do...

**Comment by Jean-PhilippeD** [comment]
ok ! 

Hope you'll find a way to handle it asap :)
I can not really help you as I'm not aware enough of enOcean specifications, unitl now I was using FHEM and it works fine without being necessary to dig in the specs
